atlasdb-client/src/main/java/com/palantir/atlasdb/persistent/rocksdb/RocksDbPersistentTimestampStore.java

```diff
@@ -136,8 +144,12 @@ private void checkNamespaceExists(StoreNamespace storeNamespace) {
     }
 
     @Override
-    public void close() {
+    public void close() throws IOException {
         rocksDB.close();
+
+        try (Stream<Path> stream = Files.walk(databaseFolder.toPath())) {
+            stream.sorted(Comparator.reverseOrder()).map(Path::toFile).forEach(File::delete);
```

Comment:
This is clever. Maybe a comment as to why the reverse comparator matters would be nice?

Reply:
Added comment explaining why the reverse lexicographical order is needed.